akka-http-avro4s/src/main/scala/de/heikoseeberger/akkahttpavro4s/AvroSupport.scala

```diff
@@ -81,13 +81,17 @@ trait AvroSupport {
         try {
           output.write(data)
         } finally {
-          // Ignoring closing exception
-          Try(output.close())
+          try output.close()
+          catch {
+            case _: IOException => // Ignoring closing exceptions
```

Comment:
Yes, you probably right. We shouldn't ignore the error, even in `close()`. I'll do an update.